input-validation/validateBearing.js

```diff
@@ -0,0 +1,6 @@
+exports.validateBearing = function validateBearing(bearing) {
+  if (typeof bearing !== "number") {
+    const ARGUMENT_TYPE = Array.isArray(bearing) ? "array" : typeof bearing;
```

Comment:
Can we make this `var` until we set up babel/typescript for folks in older environments?

Reply:
Sure, but there is const in validateNumberOfEdges already